packages/hub/src/app/users/[username]/models/[slug]/layout.tsx

```diff
@@ -14,13 +14,10 @@ export default function Layout({
   params: { username: string; slug: string };
   children: ReactNode;
 }) {
-  const url = modelRoute({ username: params.username, slug: params.slug });
-
+  const { variableName } = useParams();
   const fallback = (
     <EntityLayout
-      username={params.username}
-      slug={params.slug}
-      homepageUrl={url}
+      nodes={entityNodes(params.username, params.slug, variableName)}
```

Comment:
5 lines below there's a `height: 32`; it needs to be adjusted to match the new height in loading state.

Reply:
I'm not sure which height specifically I need to use. 50 kind of makes sense? (It's a big jump though)

<img width="2971" alt="image" src="https://github.com/quantified-uncertainty/squiggle/assets/377065/4cd8b7cc-f417-45bc-8393-e380a2227c40">

My impression from some playing with it is that there seems to be some range where it doesn't make a big difference.

Reply:
I think it should be 50 because that's what it renders to in the end. We'll have to adjust it if it changes again.

Reply:
Or 49? I see a tiny jump still.